bin/run-pacta

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+here=$(pwd)
```

Comment:
I see what's going on, but I don't like it. I'd prefer to have a shell function or script that reads some configurations (like where pacta data is) from envvars, and then calls R on the correct file, regardless of where the user currently is (`pwd`)

something that could be invoked along the lines of 
```bash
cd ~/Downloads
export PACTA_DATA=/tmp/PD202q4
run_pacta ~/portfolios/foo.csv
```